Info.plist

```diff
@@ -113,7 +113,7 @@
 	<key>SUPublicDSAKeyFile</key>
 	<string>dsa_pub.pem</string>
 	<key>SUFeedURL</key>
-	<string>http://files.openscad.org/appcast.xml</string>
+	<string>https://files.openscad.org/appcast.xml</string>
```

Comment:
We can probably also remove the `NSExceptionAllowsInsecureHTTPLoads` key as well. See history